f2/src/forms/AnonymousInfoForm.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { useLingui } from '@lingui/react'
+import { Trans } from '@lingui/macro'
+import { Form } from 'react-final-form'
+import { NextAndCancelButtons } from '../components/next-and-cancel-buttons'
+import { RadioAdapter } from '../components/radio'
+import { Stack } from '@chakra-ui/core'
+import { useStateValue } from '../utils/state'
+import { FormArrayControl } from '../components/FormArrayControl'
+
+const validate = () => {
+  return {}
```

Comment:
We need to validate by checking if `ifanonymous` is not empty. Either option must be selected.